Fatras/include/ActsFatras/Physics/EnergyLoss/BetheHeitler.hpp

```diff
@@ -22,19 +23,74 @@ namespace ActsFatras {
 /// "A Gaussian-mixture approximation of the Bethe–Heitler model of electron
 /// energy loss by bremsstrahlung" R. Frühwirth
 struct BetheHeitler {
+  using Scalar = Particle::Scalar;
+  using Vector3 = Particle::Vector3;
+
   /// A scaling factor to
   double scaleFactor = 1.;
 
+  // Simplified angle evaluation
+  bool uniformHertzDipoleAngle = false;
+
+  // electron in original state, energy loss sampled
+  Particle bremPhoton(const Particle &particle, Scalar gammaE, Scalar rndPsi,
+                      Scalar rndTheta1, Scalar rndTheta2,
+                      Scalar rndTheta3) const {
```

Comment:
Since you are drawing the random numbers separately, this function should be implemented in a `.cpp` file and not in the header.

Reply:
done